packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.stories.js

```diff
@@ -0,0 +1,140 @@
+    ]
+  },
+  {
+    favorite: true,
+    subjectId: '456',
+    locations: [
+      { 'image/jpeg': 'https://panoptes-uploads.zooniverse.org/production/subject_location/7be26d91-9dcf-4a50-a443-51ea9c630c0d.jpeg' }
+    ]
+  },
+  {
+    favorite: false,
+    subjectId: '789',
+    locations: [
+      { 'image/jpeg': 'https://panoptes-uploads.zooniverse.org/production/subject_location/8b7deee1-af99-4400-a235-2bbd21ed4c39.jpeg' }
+    ]
+  }
+]
+
+const VIDEO_SUBJECTS = [
+  {
+    favorite: false,
+    subjectId: '123',
+    locations: [
+      { 'video/mp4': 'https://panoptes-uploads.zooniverse.org/production/subject_location/279b23de-b1e8-4a1c-90c4-2d25bbee787d.mp4' }
+    ]
+  },
+  {
+    favorite: true,
+    subjectId: '456',
+    locations: [
+      { 'video/mp4': 'https://panoptes-uploads.zooniverse.org/production/subject_location/64e2df1e-830d-4126-a1bc-8f0eef6e9eb9.mp4' }
+    ]
+  },
+  {
+    favorite: false,
+    subjectId: '789',
+    locations: [
+      { 'video/mp4': 'https://panoptes-uploads.zooniverse.org/production/subject_location/d3e90135-efda-4391-b7bf-cbed262c9dc8.mp4' }
+    ]
+  }
+]
+
+function toggle () {
+  this.favorite = !this.favorite
+}
+RECENTS.forEach(recent => {
+  recent.toggleFavourite = toggle.bind(recent)
+})
+
+storiesOf('Project App / Screens / Classify / Recent Subjects', module)
+  .addDecorator(withKnobs)
+  .add('plain', () => (
+    <Grommet theme={zooTheme}>
+      <RecentSubjectsContainer
+        isLoggedIn={boolean('User logged in', true)}
+        recents={RECENTS}
+        projectName='Snapshot Serengeti'
+        slug='zooniverse/snapshot-serengeti'
+      />
+    </Grommet>
+  ))
+  .add('narrow screens', () => (
+    <Grommet theme={zooTheme}>
+      <RecentSubjectsContainer
+        isLoggedIn={boolean('User logged in', true)}
+        recents={RECENTS}
+        projectName='Snapshot Serengeti'
+        size={1}
+        slug='zooniverse/snapshot-serengeti'
+      />
+    </Grommet>
+  ), { viewport: { defaultViewport: 'iphone5' }})
+  .add('transcription', () => (
+    <Grommet theme={zooTheme}>
+      <RecentSubjectsContainer
+        isLoggedIn={boolean('User logged in', true)}
+        recents={PORTRAIT_SUBJECTS}
+        projectName='Notes from Nature'
+        slug='zooniverse/notes-from-nature'
+      />
+    </Grommet>
+  ))
+  .add('video', () => (
+    <Grommet theme={zooTheme}>
+      <RecentSubjectsContainer
+        isLoggedIn={boolean('User logged in', true)}
+        recents={VIDEO_SUBJECTS}
+        projectName='NestCams'
+        slug='spotteron/nestcams'
+      />
+    </Grommet>
+  ))
+  .add('one subject', () => (
+    <Grommet theme={zooTheme}>
+      <RecentSubjectsContainer
+        isLoggedIn={boolean('User logged in', true)}
+        recents={RECENTS.slice(0,1)}
+        projectName='NestCams'
+        slug='spotteron/nestcams'
+      />
+    </Grommet>
+  ))
```

Comment:
All of the stories need viewports set or else if you select the narrow viewport one, the other stories stay in the `iphone5` preview. The others should have `, { viewport: { defaultViewport: 'responsive' }}` to reset it.